src/main/groovy/com/github/benmanes/gradle/versions/reporter/HtmlReporter.groovy

```diff
@@ -0,0 +1,332 @@
+
+
+    printStream.println("Gradle ${gradleReleaseChannel} updates:")
+    result.gradle.with {
+      // Log Gradle update checking failures.
+      if (current.isFailure) {
+        printStream.println("<P>[ERROR] [release channel: ${CURRENT.id}] " + current.reason + "</P>")
+      }
+      if ((gradleReleaseChannel == RELEASE_CANDIDATE.id || gradleReleaseChannel == NIGHTLY.id) && releaseCandidate.isFailure) {
+        printStream.println("<P>[ERROR] [release channel: ${RELEASE_CANDIDATE.id}] " + releaseCandidate.reason + "</P>")
+      }
+      if (gradleReleaseChannel == NIGHTLY.id && nightly.isFailure) {
+        printStream.println("<P>[ERROR] [release channel: ${NIGHTLY.id}] " + nightly.reason + "</P>")
+      }
+
+      // print Gradle updates in breadcrumb format
+      printStream.print("<P>Gradle: [" + getGradleVersionUrl(running.version))
+      boolean updatePrinted = false
+      if (current.isUpdateAvailable && current > running) {
+        updatePrinted = true
+        printStream.print(" -> " + getGradleVersionUrl(current.version))
+      }
+      if ((gradleReleaseChannel == RELEASE_CANDIDATE.id || gradleReleaseChannel == NIGHTLY.id) && releaseCandidate.isUpdateAvailable && releaseCandidate > current) {
+        updatePrinted = true
+        printStream.print(" -> " + getGradleVersionUrl(releaseCandidate.version))
+      }
+      if (gradleReleaseChannel == NIGHTLY.id && nightly.isUpdateAvailable && nightly > current) {
+        updatePrinted = true
+        printStream.print(" -> " + getGradleVersionUrl(nightly.version))
+      }
+      if (!updatePrinted) {
+        printStream.print(": UP-TO-DATE")
+      }
+      printStream.println("]<P>")
+      printStream.println(getGradleUrl())
+    }
+  }
+
+  private def getGradleUrl() {
+    return "<P>For information about Gradle releases click <a target=\"_blank\" href=\"https://gradle.org/releases/\">here</a>.";
+  }
+
+  private def getGradleVersionUrl(String version) {
+    if (version == null) {
+      return "https://gradle.org/releases/";
+    }
+    return String.format("<a target=\"_blank\" href=\"https://docs.gradle.org/%s/release-notes.html\">%s</a>", version, version)
+  }
+
+  private def getUrlString(String url) {
+    if (url == null) {
+      return "";
+    }
+    return String.format("<a target=\"_blank\" href=\"%s\">%s</a>", url, url)
+  }
+
+  private def getVersionString(String group, String name, String version) {
+    String mvn = getMvnVersionString(group, name, version)
+    String bintray = getBintrayVersionString(group, name, version)
+    return String.format("%s %s %s", version, mvn, bintray)
+  }
+
+  private def getMvnVersionString(String group, String name, String version) {
+    // https://mvnrepository.com/artifact/com.azure/azure-core-http-netty/1.5.4
+    if (version == null) {
+      return "";
+    }
+    String versionUrl = String.format("https://mvnrepository.com/artifact/%s/%s/%s", group, name, version)
```

Comment:
Surprisingly this is a 3rd party site that SEOs for ad revenue, and the official one is https://search.maven.org.

Reply:
on it. similar url but not as generic.

Reply:
either is fine, just an fyi

Reply:
next commit.